Working with tests?

I love the DX of using webpack here, but I can no longer run my tests with the `meteor test` command.  I'd been using the `meteortesting:mocha` driver for 6 months without issue until I changed to using webpack. Now I get the error:

```
Error: Cannot find module '/private/var/folders/3h/cm6yv0bs7xb_7h0_47p_w36h0000gn/T/meteor-test-run1qy5334.dlt1k/node_modules/webpack'
```

Any ideas how to get this up and running again? I'm happy to contribute if you point me in the right direction!

--- a/atmosphere-packages/webpack-dev-middleware/dev-server.js
+++ b/atmosphere-packages/webpack-dev-middleware/dev-server.js
@@ -204,7 +204,7 @@ function arrangeConfig(webpackConfig) {
     return webpackConfig;
 }
 
-if (Meteor.isServer && Meteor.isDevelopment) {
+if (Meteor.isServer && Meteor.isDevelopment && !Meteor.isTest && !Meteor.isAppTest) {
     const webpack = Npm.require(path.join(projectPath, 'node_modules/webpack'))
     const webpackConfig = arrangeConfig(Npm.require(path.join(projectPath, WEBPACK_CONFIG_FILE)));
 

